client/services/SolutionBrokerService.js

```diff
@@ -18,20 +18,38 @@
  */
 
 tie.factory('SolutionBrokerService', [
```

Comment:
jw - why is it called SolutionBrokerService? Just trying to better understand the codebase.